src/meta-srv/src/region/supervisor.rs

```diff
@@ -365,8 +369,7 @@ impl RegionSupervisor {
             .collect::<Vec<_>>();
 
         for (datanode_id, region_id) in migrating_regions {
-            self.failure_detector.remove(&(datanode_id, region_id));
```

Comment:
Why do we remove this line?

Reply:
The failure detectors shouldn't be removed before failover finishes.